iotdb-core/metrics/pom.xml

```diff
@@ -31,8 +31,8 @@
     <name>IoTDB: Core: Metrics</name>
     <description>Metrics interfaces for IoTDB</description>
     <modules>
+        <module>core</module>
         <module>interface</module>
-        <module>micrometer-metrics</module>
         <module>dropwizard-metrics</module>
```

Comment:
How about just removing whole dropwizard-metrics module

Reply:
+1, and maybe there is no need to split core and micrometer. What's your opinion?